src/App.tsx

```diff
@@ -15,71 +15,93 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: `${goods[8]} is selected`,
```

Comment:
wOw, nice solution)

Reply:
looks like hardcode actually, we can change position of Jam then it will break

Reply:
also don't save whole string, it's called selectedGood so be it